pepper-apis/dsm-server/appengine/StudyManager.tmpl.yaml

```diff
@@ -13,7 +13,7 @@ network:
   name: managed
   instance_tag: study-manager
 
-entrypoint: java -javaagent:tcell/tcellagent.jar -agentpath:/opt/cprof/profiler_java_agent.so=-cprof_enable_heap_sampling=true,-cprof_service=study-manager,-cprof_cpu_use_per_thread_timers=true,-logtostderr -Xmx1640m -Dlog4j.configurationFile=log4j.xml -Dio.grpc.netty.shaded.io.netty.transport.noNative=false -jar DSMServer.jar
+entrypoint: java -javaagent:tcell/tcellagent.jar -agentpath:/opt/cprof/profiler_java_agent.so=-cprof_enable_heap_sampling=true,-cprof_service=study-manager,-cprof_cpu_use_per_thread_timers=true,-logtostderr -Xmx1640m -Dlog4j.configurationFile=log4j.xml -Dio.grpc.netty.shaded.io.netty.transport.noNative=false --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED -jar DSMServer.jar
```

Comment:
Do we want these changes in housekeeping and dss as well?  Or are there patterns there well enough established that we're not at risk of running into similar issues in those deployments?